### DISCUSSION TITLE
Direction for this repo

### DISCUSSION THREAD

**Discussion by bcheidemann** [bcheidemann/Minecraft-Overviewer-Docker#1] [Q&A]
This is currently a basic example repo. I'm going to be working on this over the next couple weeks and am open to collaborating on this/having an open discussion about how best to implement this for to make it easy for servers to use. I think this is particularly relevant as the current apt install doesn't work with current versions of Ubuntu. This is also a continuation of [this discussion](https://github.com/overviewer/Minecraft-Overviewer/issues/1992) on the original repo so check it out for some context if you're stumbling across this :D

**Comment by bcheidemann** [comment]
@wolfen351


**Copying config vs volume**
What you're saying about copying a default config into the volume makes sense to me. I suppose we could check for an existing config file in there volume first and if none exists then copy the default one.

**Clone vs copy**
Yeah it's probably the same amount of work either way tbh it's just by forking the repo I get to fix any issues as and when I have time. I think given I've done it now I will continue with this strategy 😁

**Version check**
Makes sense, I will add this step to the Dockerfile 😊

**1.17 and 1.17.1**
Gotcha. Ideal implementation would be to allow specifying this option at build time or at run time. I guess some way of doing this at runtime would be best so you don't need to rebuild the docker image every time the server is updated. Current server version could also (in theory) be detected automatically. Thoughts?

**CPU Limit and sleep**
Thanks for sharing that script - this will be very helpful for deploying this in the live environment. Also thank you for the heads up on putting `sleep` at the start of the script.

**Comment by wolfen351** [comment]
All good, I'm very excited to see how this project progresses!

**Reply by bcheidemann** [reply]
I'll keep you posted 🙂 appreciate your willingness to share your experience
